spec/features/results_download_spec.rb

```diff
@@ -12,8 +17,6 @@
   it "visits results page, clicks 'Waitlist', checks if csv file is downloaded" do
```

Comment:
Why would we try to click it? I don't think that's possible, if it's just a text basically